inc/config.class.php

```diff
@@ -421,12 +423,10 @@ public static function configUpdate($input) {
       }
 
       if (isset($_SESSION['plugin_flyvemdm_wizard_step'])) {
-         $input = static::processStep($input);
-         if (count($input) > 0 && $input !== false) {
-            static::forwardStep($input);
-         } else {
-            $input = [];
-         }
+         //disable wizard if needed
+         static::processStep($input);
```

Comment:
if you drop the argument in the method, you no longer need to pass $input here.

Initially, passing the $input is to allow some actions based on the data filled in a form embedded in the step. This would be nice to keep this ability.

Reply:
fixed